### ISSUE TITLE
pm?

### ISSUE THREAD

**Issue by BobSelf** [emrecio/icomlib#1]
How do I contact you?

**Comment by emrecio** [comment]
Yeah, no worries, not quite sure how this may work and yet be private. They always mess about email addresses here. Unknown. emrecio.bsky.social  we can take it from there? In any case, looking to do the same. Check out the define file. This kenwood project has been sitting on my desk for a bit.
